website/docusaurus.config.js

```diff
@@ -0,0 +1,317 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const path = require('path');
+
+// The top-30 locales on Crowdin are enabled
+// but we enable only a subset of those
+const locales = [
+  'en',
+  'ja',
+  //'ar',
+  //'bs-BA',
+  //'ca',
+  //'cs',
+  //'da',
+  //'de',
+  //'el',
+  'es-ES',
+  //'fa-IR',
+  //'fi',
+  //'fr',
+  //'he',
+  //'hu',
+  //'id-ID',
+  //'it',
+  //'af',
+  //'ko',
+  //'mr-IN',
+  //'nl',
+  //'no-NO',
+  //'pl',
+  //'pt-PT',
+  'pt-BR',
+  'ro',
+  'ru',
+  //'sk-SK',
+  //'sr',
+  //'sv-SE',
+  //'tr',
+  'uk',
+  //'vi',
+  'zh-Hans',
+  //'zh-Hant',
+];
+
+const localeConfigs = {
+  en: {
+    label: 'English',
+  },
+  ja: {
+    label: '日本語',
+  },
+  'es-ES': {
+    label: 'Español',
+  },
+  'pt-BR': {
+    label: 'Português (Brasil)',
+  },
+  ro: {
+    label: 'Română',
+  },
+  ru: {
+    label: 'Русский',
+  },
+  uk: {
+    label: 'Українська',
+  },
+  'zh-Hans': {
+    label: '简体中文',
+  },
+};
+
+module.exports = {
+  i18n: {
+    defaultLocale: 'en',
+    locales,
+    localeConfigs,
+  },
+  title: 'Jest',
+  titleDelimiter: '·',
+  tagline: '🃏 Delightful JavaScript Testing',
+  url: 'https://jestjs.io',
+  baseUrl: '/',
+  projectName: 'jest',
+  favicon: 'img/favicon/favicon.ico',
+  onBrokenLinks: 'warn',
+  onBrokenMarkdownLinks: 'warn',
+  presets: [
+    [
+      '@docusaurus/preset-classic',
+      {
+        docs: {
+          // homePageId: 'getting-started',
+          showLastUpdateAuthor: true,
+          showLastUpdateTime: true,
+          editUrl: 'https://github.com/facebook/jest/edit/master/docs/',
+          path: '../docs',
+          sidebarPath: './sidebars.json',
+        },
+        blog: {
+          path: path.join(__dirname, '..', 'website', 'blog'),
+          blogSidebarCount: 'ALL',
+        },
+        theme: {
+          customCss: [
+            path.resolve('src/components/v1/legacyCSS.css'),
+            path.resolve('src/css/docusaurusTheme.scss'),
+            path.resolve('src/css/algoliaDocSearchTheme.scss'),
+            path.resolve('static/css/custom.scss'),
+            path.resolve('static/css/jest.scss'),
+          ],
+        },
+      },
+    ],
+  ],
+  plugins: [
+    'docusaurus-plugin-sass',
+    [
+      '@docusaurus/plugin-client-redirects',
+      {
+        // for legacy v1 Docusaurus site: /api.html => /api
+        fromExtensions: ['html'],
+      },
+    ],
+    [
+      '@docusaurus/plugin-pwa',
+      {
+        pwaHead: [
+          {
+            tagName: 'link',
+            rel: 'icon',
+            href: 'img/jest.png',
+          },
+          {
+            tagName: 'link',
+            rel: 'manifest',
+            href: `manifest.json`,
+          },
+          {
+            tagName: 'meta',
+            name: 'theme-color',
+            content: '#15c213',
```

Comment:
get from some constant?

Reply:
Added a local variable but the only other place it is in is as a CSS variable so this basically just remove a single duplicate in the config :)